### ISSUE TITLE
loadStrings wants always a data folder?

### ISSUE THREAD

**Issue by fabianmoronzirfas** [basiljs/basil.js#303]
When I try to load a file like this:  

```js
  msgA = b.loadStrings("~/Desktop/myfile.txt");
```

I get this error msg:

<img width="420" alt="screenshot 2018-11-14 at 20 24 14" src="https://user-images.githubusercontent.com/315106/48507157-7355e380-e84b-11e8-99a1-0043bf138f16.png">

Which is wired. Happens with the latest version on master (realted to #300 ???)

**Comment by trych** [comment]
Yes, it always assumes a file name of a file in the data folder _or_ a file object.
So you could do:

```js
var myFile = file("~/Desktop/myfile.txt");
var myString = loadStrings(myFile);
```
Should we allow complete file paths as well? I think the code for detecting a complete file path vs. a data folder path is already in the `file()` function, so it would be no problem to implement.

**Comment by fabianmoronzirfas** [comment]
Not sure. For it is wired to have the data folder due to the fact that Basil is portable now. I would vote for properly resolving relative paths starting with '`./path/to/…'` and 'path/to' using 
 `File($.fileName).parent +"/path/to/…"`. It could also have a quick `File(path).exists === true` check and complain when it does not.

**Comment by trych** [comment]
> Not sure. For me it is wired to have the data folder due to the fact that Basil is portable now.

basil being portable and the data folder are not really connected though, as the data folder relates to the InDesign document, not the script file or the basil file.

> I would vote for properly resolving relative paths starting with './path/to/…' and 'path/to' using
File($.fileName).parent +"/" + path. It could also have a quick File(path).exists === true check and complain when it does not.

Directly within the `loadStrings()` function or in the `file()` function?

**Comment by fabianmoronzirfas** [comment]
Wherever a string has to point to a file

**Comment by trych** [comment]
Yes, I guess that makes sense. So I think we should implement all this in the `file()` function itself and then have the `loadStrings()` function and such as some wrapper for the `files()` function.

**Comment by trych** [comment]
Fixed in #326.
